### DISCUSSION TITLE
Video link says access denied

### DISCUSSION THREAD

**Discussion by kfedora** [microsoft/hack-together-teams#11] [Q&A] [answered]
I clicked on the video link on the readme 

https://github.com/microsoft/hack-together-teams/assets/3199282/70c9008f-8de5-4b01-9143-6db7183fb9a8

which takes me to 

https://github-production-user-asset-6210df.s3.amazonaws.com/3199282/237263956-70c9008f-8de5-4b01-9143-6db7183fb9a8.mp4

which says access denied 

```xml
<Error>
<Code>AccessDenied</Code>
<Message>Access Denied</Message>
<RequestId>HKMBHDJN47BJZMB3</RequestId>
<HostId>
6jXnPGyg769SaWmEaibUvG2o0So8jERp/emn5eAGh5cPRuQKROQySojAeLEDncr7w2HVw/baRxI=
</HostId>
</Error>
```

**Comment by waldekmastykarz** [comment]
Sorry for the trouble and thank you for bringing it up. We'll sort it out asap.

**Reply by kamalshree** [reply, marked as answer]
Hi @kfedora 

The link is fixed now you can access it here - https://github.com/microsoft/hack-together-teams/discussions/2
